prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-container/site-registration-container.component.ts

```diff
@@ -105,6 +107,24 @@ export class SiteRegistrationContainerComponent implements OnInit {
       .subscribe((updatedSite: Site) => this.updateSite(updatedSite));
   }
 
+  public onNotify(siteId: number) {
+    const data: DialogOptions = {
+      title: 'Send Email',
+      data: { 'siteId': siteId }
+    };
+
+    this.busy = this.dialog.open(SendEmailComponent, { data })
+      .afterClosed()
+      .pipe(
+        exhaustMap((result: string) => {
+          if (!result) { return EMPTY; }
```

Comment:
Could be reduced down to one line `exhaustMap((result: string) => (result) ? of(result) : EMPTY)`